### DISCUSSION TITLE
Changing ylims in Plot Footprint

### DISCUSSION THREAD

**Discussion by waltno** [GreenleafLab/ArchR#1423] [Feature Requests]
I am finding it difficult to see the whole footprint. It'd be cool to adjust the settings to fit the whole plot or be able to manually set the y limits.

What I get:
[footprints.pdf](https://github.com/GreenleafLab/ArchR/files/8665402/footprints.pdf)

**Comment by rcorces** [comment]
hmmm. thats not supposed to happen. The ylim is supposed to auto adjust . do you see this with multiple different factors or just this one?
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/Footprinting.R#L579

**Reply by waltno** [reply]
I see it with most plots I make :/ I can fork it and adjust it for my purposes

**Reply by rcorces** [reply]
I assume you dont see this with the tutorial data though right?
https://www.archrproject.com/bookdown/normalization-of-footprints-for-tn5-bias.html
